### ISSUE TITLE
Style selected answers

### ISSUE THREAD

**Issue by louiseka** [louiseka/quizzical#11]
When a user select an answer update background colour of button to #D6DBF5

Check the colour contrasts of the selected background-color and color, and adjust the background if contrast does not meet AA.
